src/Lookup/InMemoryEntityLookup.php

```diff
@@ -33,6 +37,17 @@ public function addEntity( EntityDocument $entity ) {
 	}
 
 	/**
+	 * If an exception with the same EntityId was already present it will be replaced by the new one.
```

Comment:
Indeed. It points out behaviour of this method that would otherwise require the user to look in its implementation to find out. Do you think something should be changed here?